app/models/manageiq/providers/redhat/infra_manager/vm/operations/power.rb

```diff
@@ -4,7 +4,11 @@ def validate_pause
   end
 
   def raw_start
-    with_provider_object(&:start)
+    start_with_cloud_init = custom_attributes.find_by(:name => "miq_provision_boot_with_cloud_init")
+    with_provider_object do |rhevm_vm|
+      rhevm_vm.start { |action| action.use_cloud_init(true) if start_with_cloud_init }
+    end
```

Comment:
@Fryguy Are you suggesting `Ovirt::VM#start_with_cloud_init`?  if so, then this would change to:
```ruby
with_provider_object do |rhevm_vm|
  start_with_cloud_init ? rhevm_vm.start_with_cloud_init : rhevm_vm.start
end
```
I don't think it makes the code significantly better, plus it's more complex to backport for the BZ.  I can do that in a followup if you want.